packages/tools/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -77,7 +77,7 @@ function getRealCommand(cmd, toolsRoot) {
         case "test:e2e:web:dev":
             return `cross-env DEBUG=true wdio ${join(toolsRoot, "test-config/wdio.conf.js")}`;
         case "test:e2e:web":
-            return `node ${join(toolsRoot, "scripts/e2e.js")}`;
+            return `node ${join(toolsRoot, "scripts/e2e.js")} $@`;
```

Comment:
What does this `$@` syntax do?

Reply:
Show the output :)

Reply:
This carries over all the arguments you passed from the parent command you executed